source/software/hybrid_mpi_openmp_programs.rst

```diff
@@ -254,19 +254,20 @@ Essential elements of our job script are:
 
    #! /bin/bash -l
    # Adapt nodes and ppn on the next line according to the cluster your're using!
-   #PBS -lnodes=2:ppn=20
+   #PBS -l nodes=2:ppn=20
    ...
    #
    module load foss
    #
    export OMP_NUM_THREADS=5
    #
    export OMP_PROC_BIND=true
+   
+   MPI_NUM_PROCS=$(( $PBS_NP/$OMP_NUM_THREADS ))
    #
-   mpirun -cpus-per-proc $OMP_NUM_THREADS ./hybrid_mpi
+   mpirun --np $MPI_NUM_PROCS  --map-by socket:PE=$OMP_NUM_THREADS  --bind-to core  ./hybrid_mpi
+
 
-Advanced issues
-~~~~~~~~~~~~~~~
 
 Open MPI allows a lot of control over process placement and rank
```

Comment:
Is it worth to recommend the "--report-bindings" option here?
It is quite handy for double-checking.

Reply:
It's mentioned below.